include/xgboost/learner.h

```diff
@@ -36,7 +36,8 @@ enum class PredictionType : std::uint8_t {  // NOLINT
   kContribution = 2,
   kApproxContribution = 3,
   kInteraction = 4,
-  kLeaf = 5
+  kApproxInteraction = 5,
```

Comment:
Is this a new feature? I don't see the necessary change anywhere in the C++ codebase.